src/layers.rs

```diff
@@ -281,6 +295,64 @@ impl LayersManager {
         }
     }
 
+    pub fn turn_alias_on(&mut self, name: String) {
+        if let Some(index) = self.get_idx_from_alias(name) {
+            let idx = index.clone();
```

Comment:
nitpick: I think you don't need a clone here. usize should implement `Copy`

Reply:
You get a mutable borrow reservation conflict if you don't clone. Currently a warning, but really probably should be an error. The clone almost certainly would be optimized away by compiler if it isn't necessary

Reply:
Haha gotcha. No worries then